records/benchmarks/0.17.3--after-lorem.txt

```diff
@@ -0,0 +1,396 @@
+        "relativeMarginOfError": 0.2914457565479899,
+        "standardErrorOfMean": 1.512271132946221e-7,
+        "sampleVariance": 2.264094339747022e-12
+      },
+      "completed": true,
+      "percentSlower": 0
+    },
+    {
+      "ms": 17.24137931034483,
+      "name": "copycat.sentence()",
+      "ops": 58,
+      "margin": 1.34,
+      "options": {
+        "delay": 0.005,
+        "initCount": 1,
+        "minTime": 0.05,
+        "maxTime": 5,
+        "minSamples": 5
+      },
+      "samples": 62,
+      "promise": false,
+      "details": {
+        "min": 0.01565675,
+        "max": 0.01949735425,
+        "mean": 0.01724145887096774,
+        "median": 0.0169393125,
+        "standardDeviation": 0.0009273945090137376,
+        "marginOfError": 0.000230847272031087,
+        "relativeMarginOfError": 1.3389079993677464,
+        "standardErrorOfMean": 0.00011777922042402399,
+        "sampleVariance": 8.600605753488314e-7
+      },
+      "completed": true,
+      "percentSlower": 99.41
+    }
+  ],
+  "fastest": {
+    "name": "rot13()",
+    "index": 1
+  },
+  "slowest": {
+    "name": "copycat.sentence()",
+    "index": 2
+  }
+}
+Running "10000 words" suite...
+{
+  "name": "10000 words",
+  "date": "2023-10-05T21:00:55.536Z",
+  "results": [
+    {
+      "ms": 5.0761421319796955,
+      "name": "copycat.scramble()",
+      "ops": 197,
+      "margin": 0.77,
+      "options": {
+        "delay": 0.005,
+        "initCount": 1,
+        "minTime": 0.05,
+        "maxTime": 5,
+        "minSamples": 5
+      },
+      "samples": 85,
+      "promise": false,
+      "details": {
+        "min": 0.004896162909090909,
+        "max": 0.006643333363636364,
+        "mean": 0.005066891021283422,
+        "median": 0.005035924181818182,
+        "standardDeviation": 0.00018402072841130376,
+        "marginOfError": 0.00003912130684513898,
+        "relativeMarginOfError": 0.7720968673060136,
+        "standardErrorOfMean": 0.00001995985043119336,
+        "sampleVariance": 3.386362848502682e-8
+      },
+      "completed": true,
+      "percentSlower": 64.82
+    },
+    {
+      "ms": 1.7857142857142858,
+      "name": "rot13()",
+      "ops": 560,
+      "margin": 0.78,
+      "options": {
+        "delay": 0.005,
+        "initCount": 1,
+        "minTime": 0.05,
+        "maxTime": 5,
+        "minSamples": 5
+      },
+      "samples": 96,
+      "promise": false,
+      "details": {
+        "min": 0.0017361192758620688,
+        "max": 0.0023936192413793102,
+        "mean": 0.0017856695380504813,
+        "median": 0.0017723426724137931,
+        "standardDeviation": 0.00006961359067521947,
+        "marginOfError": 0.000013925618399242639,
+        "relativeMarginOfError": 0.7798541724828908,
+        "standardErrorOfMean": 0.000007104907346552367,
+        "sampleVariance": 4.846052006697002e-9
+      },
+      "completed": true,
+      "percentSlower": 0
+    },
+    {
+      "ms": 200,
```

Comment:
At 10,000 words, it takes `200ms` for `copycat.sentence()` over the previous `166ms` unfortunately. Generating sentences this long is not something that feels like a likely use case though. That, and if it were, both these numbers aren't great.